test/torchaudio_unittest/prototype/functional/functional_test_impl.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import torch
+import torchaudio.prototype.functional as F
+from scipy import signal
+from torchaudio_unittest.common_utils import nested_params, TestBaseMixin
+
+
+class FunctionalTestImpl(TestBaseMixin):
+    @nested_params(
+        [F.convolve, F.fftconvolve],
```

Comment:
shall we split the testing by comparing `F.convolve` with `scipy.signal.convolve(..., method="direct")`, and comparing `F.fftconvolve(..., method="fft")`?